---
layout: about
title: about
permalink: /
subtitle:

profile:
  align: right 
  image: prof_pic.JPG
  image_circular: True # crops the image to make it circular
  more_info: # add address etc

news: false  # includes a list of news items
latest_posts: false # includes a list of the newest posts
selected_papers: false # includes a list of papers marked as "selected={true}"
social: true  # includes social icons at the bottom of the page
---

Hi there! I'm a postdoctoral fellow in the [Greenbaum Lab](https://www.mskcc.org/research-areas/labs/benjamin-greenbaum) and the [Balachandran Lab](https://www.mskcc.org/research-areas/labs/vinod-balachandran) at Memorial Sloan Kettering Cancer Center, where I work on a variety of problems at the intersection of cancer evolution and quantitative immunology.

Prior to this, I completed my PhD with [Benjamin Good](https://bgoodlab.github.io/) at Stanford University. My PhD research focused on quantifying the dynamics of bacterial recombination---a form of horizontal gene transfer within the same species---in the human gut microbiome. A full list of my publications is available [here](/publications/), and my Google Scholar profile can be found [here](https://scholar.google.com/citations?user=UyVL20cAAAAJ&hl=en).

<!-- Most recently, I'm interested in understanding the long-term evolution of human gut bacteria (e.g. on the timescale of early human migrations), as well as developing theoretical tools for predicting the genetic linkage between rare mutations (see our recent [Genetics paper](https://doi.org/10.1093/genetics/iyae145), in collaboration with Anastasia Lyulina). -->

<!-- In our recent [PLOS Bio paper](https://doi.org/10.1371/journal.pbio.3002472), we thoroughly quantified the dynamics of bacterial recombination (i.e. horizontal gene transfer within a species) in the human gut microbiome. Beyond answering the basic &#40;and surprisingly subtle&#41; question, "which species recombines the most?", we discovered rich patterns that shed lights on the heterogeneity of recombination rates, barriers to gene flow, and the interaction between selection and recombination, both within individual hosts and across the broader population. -->

<!-- In addition to my data explorations, I'm developing new theoretical tools for understanding how genetic linkage is shaped by the interplay between recombination, selection and genetic drift. -->

Prior to graduate school, I studied physics, math and computer science at University of Illinois at Urbana-Champaign. I was thinking about questions related to phylogenetic trees under the guidance of [Nigel Goldenfeld](https://guava.physics.ucsd.edu/~nigel/).